packages/commerce-sdk-react/src/hooks/ShopperBaskets/action.ts

```diff
@@ -0,0 +1,357 @@
+     * @see {@link https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperbaskets.shopperbaskets-1.html#removegiftcertificateitemfrombasket} for more information on the parameters and returned data type.
+     */
+    RemoveGiftCertificateItemFromBasket = 'removeGiftCertificateItemFromBasket',
+    /**
+     * Updates a gift certificate item of an existing basket.
+     * @see {@link https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-baskets?meta=updateGiftCertificateItemInBasket} for more information about the API endpoint.
+     * @see {@link https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperbaskets.shopperbaskets-1.html#updategiftcertificateiteminbasket} for more information on the parameters and returned data type.
+     */
+    UpdateGiftCertificateItemInBasket = 'updateGiftCertificateItemInBasket',
+    /**
+   * Adds new items to a basket. The added items are associated with the
+specified shipment. If no shipment id is specified, the added items are associated with the default shipment.
+Considered values from the request body, for each item are:
+
+- productId: a valid product ID. This is the ID of the product to be added to the basket. If the
+product is already in the basket, the API either increments the quantity of the existing product line item or
+creates a new product line item, based on the site preference 'Add Product Behavior'. For option products and
+product bundles containing variation masters, the API creates a new product line item regardless of the site
+preference.
+- shipmentId: a valid shipment ID (optional). This is the ID of the shipment in which the product item
+is created.
+- quantity: a number between 0.01 and 999. This is the quantity of the product to order.
+- inventoryId: a valid inventory ID (optional). This is the ID of the inventory from which the item is
+allocated.
+- bonusDiscountLineItemId: a valid bonus discount line item ID (optional). This is the ID of the
+bonus discount line item for which the added product is a selected bonus product.
+- optionItems/optionValueId: a valid option value ID. This is an option value for an option item of
+an option product.  This is only possible if the product item is an option
+product. To set option values, you must specify a collection of option items in the optionItems
+property. These option items must contain optionId and optionValueId. Also,
+the values you specify must be valid for the option product that this product item represents. Otherwise, the
+server throws an InvalidProductOptionItemException or an
+InvalidProductOptionValueItemException.
+- custom properties in the form c_\<CUSTOM_NAME\>: the custom property must correspond to a custom
+attribute (\<CUSTOM_NAME\>) defined for ProductLineItem. The value of this property must be valid for the
+type of custom attribute defined for ProductLineItem.
```

Comment:
Are we gonna keep these comments? When I look at this file, the comments seem extremely overwhelming :-o

Reply:
This is generated from the RAML specs. That's also the source of truth for the help docs (i.e. the text is identical), so we don't necessarily need to. I just included them here to demonstrate what is possible, if we decide we want that.